doc/get_started/install.rst

```diff
@@ -1,11 +1,72 @@
 
-.. include:: /links.rst
+.. .. include:: /links.rst
 
 .. _install:
 
 Install
 =======
 
+.. _install_requirements:
+
+Requirements
+------------
+
+All systems require Python 3.9 or higher on a desktop or laptop device.
+
+.. important:: Yes, this means and mobile are currently unsupported.
+
+               Please see the following to learn more:
+
+               * :ref:`Web browsers <faq_web>`
+               * :ref:`Android <faq_android>`, :ref:`iOS <faq_ios>`, and :ref:`iPad <faq_ipad>`
+
+
+Windows, Linux, and Intel Mac
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convertibles with Intel or AMD processors under 10 years old will likely work.
+
+Windows
+"""""""
+
+To avoid strange issues, install Python as follows:
+
+#. Download Python from the `official Windows download page <https://www.python.org/downloads/windows/>`_
+#. While installing, look for a checkbox marked "Add Python to PATH"
+#. When you see it, make sure it is checked before proceeding
+
+.. important:: Avoid the Microsoft Store version of Python.
+
+               It has a history of hard-to-fix bugs which will make things
+               far more difficult than they need to be.
+
+.. _requirements_mac_mseries:
+
+M-Series Macs
+"""""""""""""
+Macs with Apple's M-series processors were first introduced in 2020. These mostly work aside from
+a few issues related to window focus and unsigned applications. If something odd happens, you
+can always :ref:`ask for help. <how-to-get-help>`.
+
+.. _requirements_raspi:
+
+Raspberry Pi
+""""""""""""
+For Raspberry Pi boards:
+
+* The 4, 400, and 5 series are known to work under Linux
+* None of the Zero, Pico, 3, or any earlier models work
+
+The compatible Pi boards all support OpenGL ES 3.1 plus certain extensions. Other brands
```

Comment:
'The 4, 400, and 5 series have run Arcade with linux. The Zero, Pico,  3, or earlier can not run Arcade'

'The compatable Pi boards support OpenGL ES 3.1, which is not enough to run Arcade. They rely on OpenGL extensions to work. Other brands may also work but have not been tested.'